### ISSUE TITLE
Portfolio

### ISSUE THREAD

**Issue by araptunovich** [araptunovich/tunabits#4]
Complete transfering photos to my portfolio site and organizing them accordingly.

**Comment by saulbaizman** [comment]
Have you completed the transfer?

**Comment by araptunovich** [comment]
Yes!

**Comment by araptunovich** [comment]
I do need to add projects from the last semester and the current semester, however I do not yet have images for them. So this is something for the future.

**Comment by saulbaizman** [comment]
Yes, let's stop the content collection and formatting phase. It's often a very time-consuming phase of a project. (Content creation is even more time-consuming.)

Ready to close this issue?
